WordPress/build.gradle

```diff
@@ -150,26 +152,40 @@ android {
 
         vanilla { // used for release and beta
             dimension "buildType"
+
+            versionName versionProperties.getProperty("versionName")
+            versionCode versionProperties.getProperty("versionCode").toInteger()
+
             buildConfigField "boolean", "ME_ACTIVITY_AVAILABLE", "false"
             buildConfigField "long", "REMOTE_CONFIG_FETCH_INTERVAL", "3600"
             buildConfigField "boolean", "ENABLE_DEBUG_SETTINGS", "false"
         }
 
         zalpha { // alpha version - enable experimental features
             dimension "buildType"
+
+            versionName versionProperties.getProperty("alpha.versionName")
+            versionCode versionProperties.getProperty("alpha.versionCode").toInteger()
+
             buildConfigField "boolean", "VIDEO_OPTIMIZATION_AVAILABLE", "true"
             buildConfigField "boolean", "ENABLE_DEBUG_SETTINGS", "false"
         }
 
         wasabi { // "hot" version, can be installed along release, alpha or beta versions
             applicationIdSuffix ".beta"
             dimension "buildType"
+
+            versionName versionProperties.getProperty("alpha.versionName")
+            versionCode versionProperties.getProperty("alpha.versionCode").toInteger()
         }
 
         jalapeno { // Pre-Alpha version, used for PR builds, can be installed along release, alpha, beta, dev versions
             isDefault true
             applicationIdSuffix ".prealpha"
             dimension "buildType"
+
+            versionName versionProperties.getProperty("alpha.versionName")
+            versionCode versionProperties.getProperty("alpha.versionCode").toInteger()
```

Comment:
@AliSoftware I noticed this while reviewing #15282. [We have a `versionName` property that we pass for installable builds](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/.circleci/config.yml#L74-L79), and it looks like at some point we started to ignore that. We should probably change that `versionName` property to something like `installableBuildVersionName` and then use that for `jalapeno` variant with alpha version name as fallback. I suggest using an invalid version code either way because it's not meant to be submitted to Play Store.

Reply:
Oh shoot, good point and good catch, I will need to get that fixed!